### ISSUE TITLE
Can't publish action to the marketplace

### ISSUE THREAD

**Issue by craig-day** [dhall-lang/setup-dhall#7]
@Gabriella439 I believe you're the owner of the `dhall-lang` org. Can you help accept these terms we this action can be published to the marketplace? 

![Screen Shot 2023-01-12 at 10 07 13 AM](https://user-images.githubusercontent.com/6731804/212145486-ee340235-2575-4614-b696-55bec4bda09a.png)

**Comment by Gabriella439** [comment]
Yeah, I'm happy to accept if you can show me where to confirm that prompt
